### DISCUSSION TITLE
The equation of matrix element V3 in anharmonic self-energy session

### DISCUSSION THREAD

**Discussion by zentenmiyata** [ttadano/alamode#122] [Q&A]
Dear Tadano-sensei,

 This is Miyata. Sorry for the basic question.

On the ALAMODE theoretical background page, is the first square root of the element matrix V3 equation "ωqj" instead of "ωqn"?　 

Best regards,
Miyata,

**Comment by ttadano** [comment]
Yes, it should be ωqj. Sorry for the typo. I'll fix it.

**Reply by zentenmiyata** [reply]
Dear Tadano-sensei,

 Thank you for your reply.  

Best regards,
Miyata,
